storage_service/storage_service/settings/test.py

```diff
@@ -13,3 +13,23 @@
         "PORT": "",
     },
 }
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse'
+        }
+    },
+    'handlers': {
+        'null': {
```

Comment:
Have a console logger instead of a null logger so we can still see debugging output?  Does py.test still print logging output if the test fails with it configured like this?